### DISCUSSION TITLE
bFlash CAN log format integration

### DISCUSSION THREAD

**Discussion by RaphaelC63** [collin80/SavvyCAN#486] [Ideas]
Hi,

I'm Raphael from bFlash. I just added our CAN log file format in SavvyCAN, I would like this to be integrated.

@collin80 please find the edited files attached. 
[bFlash_log_format.zip](https://github.com/collin80/SavvyCAN/files/9308244/bFlash_log_format.zip)

For those interested bFlash is a programming, diagnostic and data acquisition tool for automotive ECU/TCU (engine and transmission control units) : https://bflash.eu/

Thanks

**Comment by collin80** [comment]
Thanks, I'll merge it in when I get a chance.

**Comment by RaphaelC63** [comment]
Great, please find attached a log example too

[example.zip](https://github.com/collin80/SavvyCAN/files/9314306/example.zip)
.
